catroid/src/main/java/org/catrobat/catroid/content/actions/LookRequestAction.kt

```diff
@@ -48,14 +48,27 @@ open class LookRequestAction : WebAction() {
     }
 
     override fun handleResponse() {
-        val look = getLookFromResponse()
-        look?.apply {
-            sprite!!.look?.lookData = this
+        val lookData = getLookFromResponse()
+        lookData?.apply {
+            insertInLookList(this)
```

Comment:
If a "Get image from" brick is not used in combination with the "Copy look" brick a new look is created, which should not be the case.